ecs-cli/modules/config/readwriter_v1.go

```diff
@@ -130,8 +133,9 @@ func (rdwr *YAMLReadWriter) Save(cliConfig *CLIConfig) error {
 
 	data, err := yaml.Marshal(cliConfig)
 	if err != nil {
-		return errors.Wrap(err, "Could not convert config to YAML")
+		return errors.Wrap(err, "Unable to parse the config")
 	}
+  // WriteFile will not change the permissions of an existing file
```

Comment:
nit: looks like this needs to be indented. not sure if it's a github formatting issue